libs/features/personalization/personalization.js

```diff
@@ -309,8 +345,10 @@ function getSelectedElement(selector, action, rootEl) {
   }
   if (checkSelectorType(selector) === 'fragment') {
     try {
-      const fragment = document.querySelector(`a[href*="${normalizePath(selector)}"]`);
-      if (fragment) return fragment.parentNode;
+      for (let i = 0; i <= 1; i += 1) {
+        const fragment = document.querySelector(`a[href*="${normalizePath(selector, !!i)}"]`);
+        if (fragment) return fragment.parentNode;
+      }
```

Comment:
Can this be simplified to:
```suggestion
const fragment = document.querySelector(`a[href*="${normalizePath(selector, false)}"], a[href*="${normalizePath(selector, true)}"]`);
if (fragment) return fragment.parentNode;
```